### ISSUE TITLE
Text renderer

### ISSUE THREAD

**Issue by thegrumpylion** [bytesparadise/libasciidoc#823]
This a feature request/suggestion to add a plain text renderer for this library.

The major use case for this is to substitute `{vars}` and remove control characters (=, |,  [...], etc.) to pass the output to an index.

From my quick research looks like this is blocked by https://github.com/bytesparadise/libasciidoc/issues/581

**Comment by xcoulon** [comment]
hello @thegrumpylion and thanks for submitting this request. I believe it's a duplicate of https://github.com/bytesparadise/libasciidoc/issues/499 but sadly I haven't had the time to work on it yet (although, it should not be too hard I believe).

As far as #581, I believe that the renderer interface is pretty stable for now. Most of the work I've been doing lately was at the parser level.

**Comment by xcoulon** [comment]
closing as a duplicate of #499
